configurations/local.template.ini

```diff
@@ -137,8 +137,8 @@ metadata_sphinx_num_of_int_fields = 10
 ;the range is used when checking service actions permissions and allowing to access certain 
 ;services without KS from the internal servers
 
-partner_otp_internal_ips = @IP_RANGE@
-otp_required_partners[] = -2
+;partner_otp_internal_ips = @IP_RANGE@
```

Comment:
Wouldn't it be better to leave this configuration and simply provide ip range that matches all and add the missing settings.remoteAddrHeaderSalt? 

what do you think?